src/plugins/intel_cpu/src/runtime_env.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cache/multi_cache.h"
+#include "config.h"
+#include "extension_mngr.h"
+#include "dnnl_scratch_pad.h"
+#include "weights_cache.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+struct RuntimeEnv {
+    typedef std::shared_ptr<RuntimeEnv> Ptr;
+
+    Config config;  // network-level config
```

Comment:
Do we really need to store a copy. M.b. a reference is just enough?

Reply:
Originally it copied into each graph, I just kept this behavior to avoid any risk of wild-pointer